pkg/certifier/osv/internal/osv_query/osv_query.go

```diff
@@ -0,0 +1,204 @@
+//
+// Copyright 2022 The GUAC Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This package is currently copied from
+// https://github.com/google/osv-scanner/blob/main/internal/osv/osv.go.
+// Currently this is not exposed by the upstream project. Once it is
+// exposed, this will be removed.
+package osv_query
```

Comment:
@oliverchang does OSV have a go library that we can use?

Reply:
Sorry for the **very** late reply! 

We recently did some refactoring and now there's a bunch of public package available here: https://github.com/google/osv-scanner/tree/main/pkg

Are these usable here?

Reply:
@oliverchang I looked through the packages but primarily I create a bunch of batchedQueries based on the purls and make the request to OSV endpoint. Is there a way to expose that for use?